### ISSUE TITLE
dbToGain accuracy

### ISSUE THREAD

**Issue by Idicious** [Tonejs/Tone.js#278]
Looking through the sourcecode I found that the Tone.dbToGain method is not a reverse of Tone.gainToDb
The values are very close but I was wondering if there is a reason for this? When changing it these tests failed by a tiny fraction (0.000...1): **'TransportTimelineSignal can automate values with different units'** and **'TimelineSignal can automate values with different units'**

Current Tone.dbToGain:
```
Tone.dbToGain = function(db) {
		return Math.pow(2, db / 6);
	};
```

Suggested Tone.dbToGain (exact reverse of gainToDb):
```
Tone.dbToGain = function(db) {
		return Math.pow(10, db / 20);
	};
```

**Comment by tambien** [comment]
Good catch. Not sure where i took that formula from. Do you (or anyone) have a citation or reference for amplitude <-> db conversion formulas? It'd be good if both were consistent and also correct. 

Also happy to take a PR if you've got a moment.

**Comment by Idicious** [comment]
The gain to dB source can be found at [](https://en.wikipedia.org/wiki/Decibel#Conversions) 
The best source I could find with the reverse stated is: [](http://nl.mathworks.com/help/signal/ref/db2mag.html)

I'll make a pull request for this, I still have to figure out the correct values for the test cases.

**Comment by Idicious** [comment]
After playing around with the values in the tests a bit I believe your reverse calculation is more accurate while ramping values, and the difference is so tiny I think the current one is more accurate when taking ramping into account.
It seems that during ramping it at times under or overshoots the value, and the current implementation seems to take that into account somehow.

**Comment by Idicious** [comment]
I've made a pull request with the changes to Tone.dbToGain and the associated tests that were failing.

**Comment by tambien** [comment]
Fixed by #279
